src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -44413,11 +44407,11 @@ static void Main()
             var model = comp.GetSemanticModel(tree);
             var declarators = tree.GetRoot().DescendantNodes().OfType<VariableDeclaratorSyntax>().ToArray();
             var symbol = (LocalSymbol)model.GetDeclaredSymbol(declarators[0]);
-            // https://github.com/dotnet/roslyn/issues/26198 public API should show inferred nullability
-            Assert.Equal("System.String", symbol.TypeWithAnnotations.ToTestDisplayString(true));
-            Assert.Equal(NullableAnnotation.Oblivious, symbol.TypeWithAnnotations.NullableAnnotation);
+            Assert.Equal("System.String?", symbol.TypeWithAnnotations.ToTestDisplayString(true));
+            Assert.Equal(NullableAnnotation.Annotated, symbol.TypeWithAnnotations.NullableAnnotation);
             symbol = (LocalSymbol)model.GetDeclaredSymbol(declarators[1]);
             Assert.Equal("System.Int32", symbol.TypeWithAnnotations.ToTestDisplayString(true));
+            // https://github.com/dotnet/roslyn/issues/34412 tracks standardizing the representation of value types
```

Comment:
>// https://github.com/dotnet/roslyn/issues/34412 tracks standardizing the representation of value types [](start = 12, length = 103)

When Chuck and you merge your PRs, this will be a conflict. The second person to commit should make sure to re-run CI and remove the comment